js/editor-libs/console-utils.js

```diff
@@ -74,7 +74,9 @@ module.exports = {
      */
     formatOutput: function(input) {
         'use strict';
-        if (typeof input === 'string') {
+        if (typeof input === 'undefined') {
```

Comment:
Same here, we can simply use `if (typeof input === undefined) {`

Reply:
Sure.